pilot/pkg/model/service.go

```diff
@@ -29,6 +29,8 @@ import (
 	"strconv"
 	"strings"
 
+	"time"
```

Comment:
delete newline above this (to merge the stdlib import sections)

Reply:
done